pyproject.toml

```diff
@@ -54,7 +54,7 @@ dependencies = [
     "sendgrid>=6.4.8,<6.5.0",
     "sentry-sdk~=2.0",
     "slack_sdk>=3.10,<4.0",
-    "sretoolbox~=2.6",
+    "sretoolbox~=2.7",
```

Comment:
let's pin the exact version `==2.7.0`

Reply:
done in https://github.com/app-sre/qontract-reconcile/pull/5224/commits/bb107bdb5530097311fa27521f2a6e0d158e4ef9